### DISCUSSION TITLE
Local Stable diffusion through API?

### DISCUSSION THREAD

**Discussion by NovNovikov** [LostRuins/koboldcpp#4] [Q&A] [answered]
Hi I really like your small project here. I noticed that the Kobold lite interface has support for image generation, which is great, however the question comes - can I use the local running stable diffusion through API instead of sending the request to the Horde like it is implemented in the Kobold AI(non-lite)?

**Comment by NovNovikov** [comment, marked as answer]
I found out that it is possible if I connect the non-lite Kobold AI to the API of llamaccp for Kobold. It is done by loading a model -> online sources -> Kobold API and there I enter localhost:5001

**Reply by LostRuins** [reply]
Hi, Sorry I missed this question earlier. You are correct - currently Kobold Lite does not support using Stable Diffusion locally, but the main Kobold client does. So you can use that and connect it to your locally running llamacpp-for-kobold instance.

**Comment by LostRuins** [comment]
Closing - KoboldCpp supports local image generation now.
